web/src/components/onboarding/ReturnToPreviousBusinessBar.tsx

```diff
@@ -0,0 +1,60 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { Icon } from "@/components/njwds/Icon";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getNavBarBusinessTitle } from "@/lib/domain-logic/getNavBarBusinessTitle";
+import { removeBusiness } from "@/lib/domain-logic/removeBusiness";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { templateEval } from "@/lib/utils/helpers";
+import Config from "@businessnjgovnavigator/content/fieldConfig/config.json";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+
+interface Props {
+  previousBusiness: Business | undefined;
+}
+
+export const ReturnToPreviousBusinessBar = (props: Props): ReactElement | null => {
+  const { updateQueue, userData } = useUserData();
+  const router = useRouter();
+  const businessName = getNavBarBusinessTitle(props.previousBusiness);
+
+  if (!props.previousBusiness) return null;
```

Comment:
[dust] should this be a new pattern we follow (returning null instead of react fragments)?

Reply:
I don't think it matters? React Fragments don't get inserted into the DOM, and neither do null values.

Reply:
Do you think this is something we should make consistent?

Reply:
I don't think there is a need to make it consistent.